testing/test_windows_subst.py

```diff
@@ -0,0 +1,95 @@
+import os.path
+import subprocess
+import sys
+from string import ascii_lowercase
+
+from py._path.local import LocalPath
+
+import pytest
+from _pytest import pytester
+from _pytest.fixtures import FixtureRequest
+
+
+class _SubstTmpdirFactory:
```

Comment:
```suggestion
class SubstTmpdirFactory:
```

No need to use `_` here, this class is already local to the module.